.github/CODEOWNERS

```diff
@@ -1,19 +1,30 @@
+# Code generated by githubgen. DO NOT EDIT.
 #####################################################
 #
-# List of approvers for OpenTelemetry Collector
+# List of codeowners for OpenTelemetry Collector
 #
 #####################################################
 #
 # Learn about membership in OpenTelemetry community:
-#  https://github.com/open-telemetry/community/blob/main/community-membership.md
-# 
+# https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md
 #
-# Learn about CODEOWNERS file format: 
-#  https://help.github.com/en/articles/about-code-owners
+#
+# Learn about CODEOWNERS file format:
+# https://help.github.com/en/articles/about-code-owners
 #
 
 * @open-telemetry/collector-approvers
 
 # List all maintainers individually so all of them are notified
 .github/CODEOWNERS @codeboten @BogdanDrutu @dmitryax @mx-psi
-docs/rfc           @codeboten @BogdanDrutu @dmitryax @mx-psi
+
+cmd/githubgen/                         @open-telemetry/collector-approvers @atoulme
```

Comment:
hope this fine since @atoulme is also maintainer for the tool in contrib :)

Reply:
feel free to put your name here too.